packages/frontity-org-theme/src/processors/homepage/need-inspiration.tsx

```diff
@@ -0,0 +1,42 @@
+import { Processor } from "@frontity/html2react/types";
+import { css } from "frontity";
+import React from "react";
+
+import FrontityOrg from "../../../types";
+
+export const needInspirationSection: Processor<
+  React.HTMLProps<HTMLElement>,
+  FrontityOrg
+> = {
+  name: "need-inspiration-section",
+  test: ({ node }) =>
+    node.type === "element" &&
+    node.props?.className?.split(" ").includes("need-inspiration-section"),
+  processor: ({ node }) => {
+    if (node.type !== "element") {
+      return node;
+    }
+
+    //Style the section
+    node.props.css = css`
+      ${node.props.css};
+      > div {
+        padding: 100px 0px 0px;
```

Comment:
The parent div is a `.wp-group-group.has-background` and so it has `padding: 20px 30px;` by default. Did you have that in mind?

Reply:
You're right. I think this should be handled by #56